Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -329,6 +329,8 @@ To complete our example :file:`composer.json`, it would look like this:
             ]
         }
     }
+    
+After adding files to the autoload you should run `composer dumpautoload`. This command will re-generate the vendor/autoload.php file
```

Comment:
`composer dump-autoload`, right?

Reply:
Both are fine.

Reply:
Yeah I just tried both commands and they both work in my terminal